### DISCUSSION TITLE
GitHub Mobile iOS quick action notifications on 2FA

### DISCUSSION THREAD

**Discussion by msingstad** [community/community#38053] [Mobile] [answered]
Should the 2FA/MFA notifications on GitHub Mobile include iOS quick actions? Would have been great in my case.
https://developer.apple.com/documentation/usernotifications/declaring_your_actionable_notification_types

**Comment by stevepopovich** [comment, marked as answer]
Hi Martin, this is great feature request. I will pass it on to the team!
